package.json

```diff
@@ -28,12 +28,21 @@
   "homepage": "https://github.com/pea3nut/short-night#readme",
   "devDependencies": {
     "@types/moveto": "^1.8.0",
-    "node-sass": "^4.11.0",
+    "@types/node": "^17.0.35",
```

Comment:
Where is the `package-lock.json`? If you update the `package.json`, I think `package-lock.json` will be also updated accordingly and automatically.

Reply:
package-lock.json added.